simplq/src/components/common/QueueHistory/HistoryRow.js

```diff
@@ -36,19 +37,19 @@ export default (props) => {
   //for medium-sized to larger-sized devices, function returns a larger icon
   const getIcon = () => {
     const fontSize = "180%";
-    return props.data.action === 'ADDED'
+    return props.data.eventType === 'TOKEN_ADDED'
       ? <AddIcon  style={{fontSize: fontSize}} />
-      : props.data.action === 'REMOVED'
+      : props.data.eventType === 'TOKEN_REMOVED'
```

Comment:
Post merge comment: I would have preferred a if else here for readability over a complex ternary if operator. 😄

Reply:
![image](https://user-images.githubusercontent.com/11256376/132891834-20bbc188-7967-4a8b-85a0-4abb5628a92a.png)

Reply:
@daltonfury42 , you are right I also do prefer if-else. Idk why the hell I did a ternary there. Will surely change it to an if-else, and update the ui also of the `/scanQr` route in my next PR